src/components/ProductList/ProductList.component.scss

```diff
@@ -0,0 +1,3 @@
+.main-content {
+	padding-top: 50px !important;
```

Comment:
It's not good practice to abuse the `!important` CSS rule. It's better to add an additional class to the element you are wanting to customize and increase the [selector specificity](https://css-tricks.com/specifics-on-css-specificity/) within the CSS.

Reply:
removed !important from CSS